net.launchpad.qpdfview.yml

```diff
@@ -0,0 +1,156 @@
+app-id: net.launchpad.qpdfview
+default-branch: stable
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: "6.7"
+command: qpdfview
+copy-icon: true
+rename-icon: qpdfview
+rename-desktop-file: qpdfview.desktop
+rename-appdata-file: qpdfview.appdata.xml
+
+finish-args:
+  - --filesystem=host:ro # open file dialog; drag & drop
+  - --device=dri # hardware acceleration
+  - --share=ipc # x11
+  - --socket=wayland # gui
+  - --socket=fallback-x11 # gui
+  - --socket=cups # printing
+  - --own-name=local.qpdfview # unique instance
```

Comment:
Does the app really use `.local` ?

Reply:
this is a bug if that's the case.

Reply:
This should be fixed upstream. You should probably open an issue